src/styles/00-base/02-typography/01-typefaces/_typefaces.scss

```diff
@@ -1,4 +1,2 @@
 // Typography Variables
-$font-body: 'HelveticaNeue-Light', 'Helvetica Neue', Helvetica, Arial,
-  'Lucida Grande', sans-serif;
-$font-heading: 'HelveticaNeue-Light', 'Arial', sans-serif;
+@import '../../node_modules/system-font-css/system-font.scss';
```

Comment:
I pushed up a fix that adds those two variables back in—unfortunately we can't delete them even though they reference the same font-stack as they're used somewhere 😬

Reply:
It all looks good, yeah, I only realized after I pushed that the font thing wasn't working o.o. 
Anyway I'll push.